.github/workflows/main.yml

```diff
@@ -136,4 +136,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/setup
       - name: Lint Swift
-        run: bundle exec rake lint:swift
\ No newline at end of file
+        run: bundle exec rake lint:swift
+
+  embedded-libraries:
+    name: "Lint Embedded Libraries"
+    runs-on: macos-13
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup
+      - name: Lint Embedded Libraries
+        run: bundle exec rake lint:EmbeddedLibraries
```

Comment:
Added a new CI job that lints the `Sources/Private/EmbeddedLibraries` directory and verifies that it doesn't have any `public` symbols. Confirmed that it fails as expected if there's an unexpected `public` symbol:

<img width="1460" alt="Screenshot 2023-06-26 at 3 25 02 PM" src="https://github.com/airbnb/lottie-ios/assets/1811727/f13e4d26-0694-4ae8-9e89-4505bde630ed">